lib/stat_tracker.rb

```diff
@@ -3,21 +3,23 @@
 require_relative './game_teams_collection'
 require 'csv'
 
-#THE PARENT
 class StatTracker
   attr_reader :games, :teams, :game_teams
 
   def initialize(locations)
-    @games = GamesCollection.new(locations[:games], self)
     @teams = TeamsCollection.new(locations[:teams], self)
+    @games = GamesCollection.new(locations[:games], self)
     @game_teams = GameTeamsCollection.new(locations[:game_teams], self)
-    # @seasons_collection = SeasonsCollection.new(self)
   end
 
   def self.from_csv(locations)
     self.new(locations)
```

Comment:
This is definitely functional but also completely negates the point of doing .from_csv instead of just .new. This is low priority but I would like to think about how how we might be able to leverage .from_csv to do something that .new can't